### ISSUE TITLE
Please publish the master branch to hex.pm.

### ISSUE THREAD

**Issue by toddholmberg** [davepersing/ex_aws_ecs#2]
Hello,

Is it possible to publish the lastest master branch to hex.pm? The published version still requires ExAws "2.0.0". This messes up dependency resolution:

```bash
Failed to use "ex_aws" (version 2.1.3) because
  ex_aws_cloudwatch (version 2.0.4) requires ~> 2.0
  ex_aws_ec2 (version 2.0.1) requires ~> 2.0
  ex_aws_ecs (version 0.1.1) requires ~> 2.0.0
  ex_aws_sqs (version 3.2.1) requires ~> 2.0
  mix.lock specifies 2.1.3
```

Thanks!

**Comment by bigboxdave** [comment]
@toddholmberg  - I'm really sorry for the delay.  Apparently I didn't have notifications enabled for my own repo.  🤦‍♂️ 

I'll work on getting the package republished today or tomorrow.  Thanks for your patience!

**Comment by davepersing** [comment]
@toddholmberg - I've published version 0.1.2 to Hex with the updates.

**Comment by toddholmberg** [comment]
Thanks, @davepersing !
